Kernel/Interrupts/PIC.cpp

```diff
@@ -161,7 +161,7 @@ void PIC::remap(u8 offset)
     IO::out8(PIC0_CTL, ICW1_INIT | ICW1_ICW4);
     IO::out8(PIC1_CTL, ICW1_INIT | ICW1_ICW4);
 
-    /* ICW2 (upper 5 bits specify ISR indices, lower 3 idunno) */
+    /* ICW2 (upper 5 bits specify ISR indices, lower 3 unspecified) */
```

Comment:
That's a weird formulation. How about "`upper 5 bits specify ISR indices, lower 3 don't specify anything`"?